crates/db_schema/src/impls/tagline.rs

```diff
@@ -44,12 +44,13 @@ impl Tagline {
   async fn clear(conn: &mut AsyncPgConnection) -> Result<usize, Error> {
     diesel::delete(tagline).execute(conn).await
   }
-  pub async fn get_all(pool: &DbPool, for_local_site_id: LocalSiteId) -> Result<Vec<Self>, Error> {
+  pub async fn get_all(pool: &DbPool, for_local_site_id: LocalSiteId) -> Result<Option<Vec<Self>>, Error> {
```

Comment:
Don't do this here, just return a `Result<Vec<Self>> . Where you call it, do `.await.ok()` to turn the result into an option.

Reply:
Fixed